src/features/game/expansion/Land.tsx

```diff
@@ -601,7 +601,7 @@ const getIslandElements = ({
 
   {
     clutter &&
-      isVisiting &&
+      // isVisiting &&
```

Comment:
<img width="552" height="209" alt="Screenshot 2025-08-03 at 1 13 49 PM" src="https://github.com/user-attachments/assets/e144129a-66e1-4117-85d9-19f648deaa89" />

When playing on your own island, this was throwing an error.

Reply:
Oop, this wasn't supposed to be here